SlevomatCodingStandard/Sniffs/Commenting/ForbiddenCommentsSniff.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Commenting;
+
+use SlevomatCodingStandard\Helpers\DocCommentHelper;
+
+class ForbiddenCommentsSniff implements \PHP_CodeSniffer\Sniffs\Sniff
+{
+
+	public const CODE_COMMENT_FORBIDDEN = 'CommentForbidden';
+
+	/** @var string[] */
+	public $forbiddenCommentPatterns = [];
+
+	/**
+	 * @return mixed[]
+	 */
+	public function register(): array
+	{
+		return [
+			T_DOC_COMMENT_OPEN_TAG,
+		];
+	}
+
+	/**
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint
+	 * @param \PHP_CodeSniffer\Files\File $file
+	 * @param int $pointer
+	 */
+	public function process(\PHP_CodeSniffer\Files\File $file, $pointer): void
+	{
+		$comments = DocCommentHelper::getDocCommentDescription($file, $pointer);
+
+		if ($comments === null) {
+			return;
+		}
+
+		foreach ($this->forbiddenCommentPatterns as $forbiddenCommentPattern) {
+			foreach ($comments as $comment) {
+				if (preg_match($forbiddenCommentPattern, $comment) === 0) {
+					continue;
+				}
+
+				$file->addError(
```

Comment:
Could you try to make it fixable? I think it could be easy:

1. Get the whole doccomment and use `preg_replace` with the pattern.
2. Replace the whole doccomment.

Nice to have:
- Remove the doccomment if it's empty.
- Remove empty lines before first annotation.

Reply:
I don't think I'll be able to provide it here, it's not trivial. At least I added Comment value object holding pointer information so it's a bit easier to implement.

Reply:
Ok, I'll try to do it myself later.

Reply:
Btw maybe I'd consider separate sniff for fixing empty doc comments or multiple/any (configurable) empty doc comment lines. There are multiple cases where some other sniffs leave out empty `*` line before end tag.